attest/core/src/ias/verify.rs

```diff
@@ -561,9 +561,11 @@ pub struct VerificationReport {
     /// DER-formatted bytes, from the X-IASReport-Signing-Certificate HTTP
     /// header.
     #[prost(bytes, repeated)]
+    #[digestible(never_omit)]
```

Comment:
I think this might not be needed, I think `Vec<Vec<u8>>` is going to match to `Vec<T>` and we didn't change the behavior of that in this commit

Reply:
Good catch